web_app/src/form.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@font-face{
+	font-family: 'Raleway', sans-serif;
+}
+
+.modal{
+	font-family: 'Raleway';
+	display: none;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0,0,0,0.4);
+}
+.modal-content{
+	display: flex;
+	justify-content: center;
+}
+
+#title{
+	padding-left: 100px;
+	padding-right: 100px;
+	padding-top: 10px;
+	padding-bottom:10px;
+	text-align: center;
+	color: #fff;
+	background-color: #28493a;
+	margin:0 auto;
+}
+
+#form-body{
+	padding-top: 10px;
+	padding-bottom: 10px;
+	background-color: #fff;
+}
+
+#text-input{
+	width: 100px;
+}
+
+.desc{
```

Comment:
I personally prefer the full `description` name, in my mind there's no benefit to shortening names like this, which can lead to confusion in many cases of name shortening. I'd say typing it out shouldn't be an issue with a good editor with autocomplete, and I'd imagine that we could minify it pre-production if we wanted to.